docs/docs/components/reanimated_swipeable.md

```diff
@@ -0,0 +1,219 @@
+---
+id: reanimated_swipeable
+title: Reanimated Swipeable
+sidebar_label: Reanimated Swipeable
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import GifGallery from '@site/components/GifGallery'
+
+<GifGallery>
+  <img src={useBaseUrl("gifs/sampleswipeable.gif")} height="120" />
+</GifGallery>
+
+:::info
+This component is a drop-in replacement for the `Swipeable` component, rewritten using `Reanimated`.
+:::
+
+Reanimated `Swipeable` allows for implementing swipeable rows or similar interaction. It renders its children within a panable container allows for horizontal swiping left and right. While swiping one of two "action" containers can be shown depends on whether user swipes left or right (containers can be rendered by `renderLeftActions` or `renderRightActions` props).
+
+### Usage:
+
+To use it, import it in the following way:
+
+```js
+import Swipeable from 'react-native-gesture-handler/ReanimatedSwipeable';
+```
+
+## Properties
+
+### `friction`
+
+a number that specifies how much the visual interaction will be delayed compared to the gesture distance. e.g. value of 1 will indicate that the swipeable panel should exactly follow the gesture, 2 means it is going to be two times "slower".
```

Comment:
nitpick: any reason to start sentences with small letters rather than capital ones?
rest of the documentation uses capital letters I believe

Reply:
No reason at all, fixing.